MIP/move-bridge-architecture/README.md

```diff
@@ -0,0 +1,165 @@
+# MIP-\<number\>: MOVE Token -- Bridge Design
+
+- **Description**: Architecture of the bridge for Move token.
+- **Authors**: [Franck Cassez](mailto:franck.cassez@movementlabs.xyz)
+- **Desiderata**: [MIP-\<number\>](../MIP/mip-\<number\>)
+
+## Abstract
+
+This MIP describes the high-level architecture of the MOVE token bridge. The architecture describes the main bridge components and high-level requirements.
+
+## Motivation
+
+The Movement chain (L2) uses the \$MOVE token to pay for gas fees. As a result users need to hold \$MOVE tokens to pay for their transactions.
+The _native_ \$MOVE token is an ERC-20 contract on Ethereum (L1).  By native, we mean that this is the location where the token is minted and burned and where the total supply is set and possibly modified (inflation/deflation). The \$MOVE token reserve is in the L1 contract.
+
+To use the Movement chain and pay for gas fees, a user will acquire \$MOVE (native) tokens on L1, and bridge them to L2. On the L2 they can the token to pay for gas fees or with any other dApps that transact the \$MOVE token.
+A user can choose to migrate their L2 \$MOVE tokens back to the L1 at any time.
+Cross-chain transfers are usually done through a bridge.
+
+### A standard bridge architecture
+
+The process of transferring tokens across different chains is implemented with a _bridge_ (between the chains).
+
+There are several choices for the architecture of a bridge, and we describe here a classical bridge with a  _lock-mint_ protocol (see Chainlink's [What Is a Cross-Chain Bridge?](https://chain.link/education-hub/cross-chain-bridge) for a quick introduction to types of bridges).
+
+In the sequel, we use
+
+- **L1\$MOVE** for the native token on L1,
+- **L2\$MOVE** for the _wrapped_ token on L2.
+
+> [!WARNING]  
+> The transfer of tokens is one-to-one: a user bridges $k$ L1\$MOVE tokens to L2, and they receive $k$ L2\$MOVE tokens. Same one-to-one ratio applies from L2 to L1.  The bridge does not allow for _swapping_ tokens.
```

Comment:
1) 
> The bridge does not allow for _swapping_ tokens.

can you define swapping

2) 
> The same one-to-one ratio applies from L2 to L1.  

it seems this would be burn-and-release instead of a mint-and-lock.. is there such terminology in literature?

Reply:
so according to https://chain.link/education-hub/cross-chain-bridge, it is a lock-and-mint if I am correct.

Reply:
swapping would be exchanging $MOVE for another token e.g. USDT.

Reply:
according to that link the reverse direction is called "burn and unlock", while the entire bridge design is called "lock and mint". 

i am now suggesting to at this here (Line 43) 
https://github.com/movementlabsxyz/MIP/pull/43/files